keras_rcnn/layers/losses.py

```diff
@@ -0,0 +1,71 @@
+import keras.layers
+
+
+class Classification(keras.layers.Layer):
+    def __init__(self, anchors, **kwargs):
+        self.anchors = anchors
+
+        self.is_placeholder = True
+
+        super(Classification, self).__init__(**kwargs)
+
+    def _loss(self, y_true, y_pred):
+        # Binary classification loss
+        x, y = y_pred[:, :, :, :], y_true[:, :, :, self.anchors:]
+
+        a = y_true[:, :, :, :self.anchors] * keras.backend.binary_crossentropy(x, y)
```

Comment:
I have two questions here, why use `binary_crossentropy`? And why multiply with `y_true`?
I'm thinking `binary_crossentropy` should be used when there is only one output, but we have two. One option is to use only one of the two outputs (output for foreground, probably) and use `binary_crossentropy` (the option you seem to implement here), the other option is to use `categorical_crossentropy` and convert the labels to one_hot vectors. Which has the preference? I don't know :) intuitively I'd guess `categorical_crossentropy`. I'd love to hear your thoughts about this.